switch_numbers.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+
+
+def replace_numbers(array, number: int, action: chr):
+    """
+    This function replaces all numbers in a given array which is equal, less and greater to a given number.
+    :param array: array of random numbers
+    :param number: number to look for in the array
+    :param action: which numbers to replace (smaller, grates or equal to number)
+    :return: array
+    """
+    replace_with = np.pi
+    if action == '<':
+        return np.where(array < number, replace_with, array)
```

Comment:
This action doesn't replace the indicated spaces; it returns a matrix with 'True' and 'False' values for the entire matrix.